runtime/cudaq/platform/default/rest/RemoteRESTQPU.cpp

```diff
@@ -43,6 +43,7 @@
 #include <fstream>
 #include <iostream>
 #include <netinet/in.h>
+#include <openssl/evp.h>
```

Comment:
Would `llvm/Support/Base64.h` work here? I don't think it matters since both are lying around waiting to be used (LLVMSupport and libssl)

Reply:
Yes, I believe so. (Not sure how I missed that option.) Updated in 848ea172e.